docs/sphinx_doc/en/source/tutorial/210-rag.md

```diff
@@ -190,6 +190,109 @@ RAG agent is an agent that can generate answers based on the retrieved knowledge
 Your agent will be equipped with a list of knowledge according to the `knowledge_id_list`.
 You can decide how to use the retrieved content and even update and refresh the index in your agent's `reply` function.
 
+## (Optional) Setting up a local embedding model service
+
+For those who are interested in setting up a local embedding service, we provide the following example based on the
+`sentence_transformers` package, which is a popular specialized package for embedding models (based on the `transformer` package and compatible with both HuggingFace and ModelScope models).
+In this example, we will use one of the SOTA embedding models, `gte-Qwen2-7B-instruct`.
+
+* Step 1: follow the instruction on [HuggingFace](https://huggingface.co/Alibaba-NLP/gte-Qwen2-7B-instruct) or [ModelScope](https://www.modelscope.cn/models/iic/gte_Qwen2-7B-instruct ) to download the embedding model.
```

Comment:
==> Follow (upper case)

Reply:
Maybe we can add guidance for huggingface mirror by 
```bash
export HF_ENDPOINT=https://hf-mirror.com
```
or 
```python
os.environ["HF_ENDPOINT"] = "https://hf-mirror.com"
```

Reply:
updated